app/controllers/subscribers_controller.rb

```diff
@@ -11,16 +11,18 @@ def subscriptions
   end
 
   def change_address
-    if subscriber_params[:on_conflict] == "merge"
-      Subscriber.transaction do
+    Subscriber.transaction do
```

Comment:
Interesting what is the thinking behind moving the if statement into the transaction?
I'm guessing it still does nothing if the `if` doesn't fire, but will rollback if the call doesn't work?

Reply:
It's so that the `subscriber.update` on line 23 happens in the same transaction as the merging